src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy

```diff
@@ -597,6 +597,66 @@ class PartialUpdateSpec extends HealthCheckSpecification {
         flow && flowHelperV2.deleteFlow(flow.flowId)
     }
 
+    @Tidy
+    @Tags(LOW_PRIORITY)
+    def "Able to update a flow endpoint for a single-swithc flow using partial update"() {
+        given: "An active single-switch flow"
+        def swPair = topologyHelper.singleSwitchPair
+        def flow = flowHelperV2.randomFlow(swPair)
```

Comment:
'randomFlow' should only be used when endpoint switches are different. You should use `singleSwitchFlow` here, it guarantees no unexpected conflicts for a single-sw flow
```suggestion
        def flow = flowHelperV2.singleSwitchFlow(topology.activeSwitches[0])
```

Reply:
fixed